com.unity.shadergraph/Editor/GraphUI/DataModel/ShaderGraphModel.cs

```diff
@@ -22,7 +22,7 @@ public class ShaderGraphModel : GraphModel
 
         // TODO: This will eventually delegate to real serialized data. For now, it's kept here ephemerally. Until
```

Comment:
Does it make sense for the comment to go away with this change?

Reply:
Probably yeah, I wanted to do a cleanup pass after I sort out the crash and other issues, but I think we're good for save/load now. Not sure about assembly reload but will modify this comment to match our current state.